chart/open-feature-operator/values.yaml

```diff
@@ -33,8 +33,17 @@ sidecarConfiguration:
   debugLogging: false
   ## @param sidecarConfiguration.otelCollectorUri Otel exporter uri.
   otelCollectorUri: ""
-  ## @param sidecarConfiguration.resources Override resources of the flagd sidecar.
-  resources: { }
+  resources:
+    limits:
```

Comment:
This would remove support for Resource Claims :( 
https://kubernetes.io/docs/concepts/scheduling-eviction/dynamic-resource-allocation/

See the types here: https://github.com/kubernetes/api/blob/master/core/v1/types.go#L2604C11-L2604C24